src/NHibernate/Driver/DriverBase.cs

```diff
@@ -313,9 +313,15 @@ public DbParameter GenerateOutputParameter(DbCommand command)
 
 		public virtual bool RequiresTimeSpanForTime => false;
 
+#if NETCOREAPP2_0
+		public virtual bool SupportsSystemTransactions => false;
```

Comment:
System transactions are officially in .Net Core 2.0 according to [documentation](https://docs.microsoft.com/en-us/dotnet/api/system.transactions.transactionscope).
But due to the modular nature of .Net Core, it seems its support is still lacking from data providers as stated [here](https://github.com/dotnet/corefx/issues/2949#issuecomment-316480951) or [here](https://github.com/dotnet/corefx/issues/2949#issuecomment-326523019).

Is that the reason for disabling `System.Transactions` globally? It looks like it could be still supported within some data providers.

Reply:
These properties are used only in tests, so I intend to leave them as is.